packages/hledger/build.sh

```diff
@@ -0,0 +1,52 @@
+# http(s) link to package home page.
```

Comment:
Remove unnecesary comments for `TERMUX_*` variables

Reply:
done